superset/reports/notifications/base.py

```diff
@@ -21,11 +21,13 @@
 import pandas as pd
 
 from superset.reports.models import ReportRecipients, ReportRecipientType
+from superset.utils.core import HeaderDataType
 
 
 @dataclass
 class NotificationContent:
     name: str
+    header_data: HeaderDataType  # this is optional to account for error states
```

Comment:
@AAfghahi your comment doesn't seem to reflect the actual type and default, i.e., `header_data` is required.

Reply:
@AAfghahi it looks like this value is always being passed into the class. Can we just remove the comment or is there some other intention here?